pkg/manager/options.go

```diff
@@ -49,10 +49,6 @@ type Options struct {
 	//
 	// Defaults to the eponymous constant in this package.
 	MaxConcurrentReconciles int
```

Comment:
```suggestion
	MaxConcurrentReconciles int

```
nit

Reply:
Also looks like we should fix the godoc comment of PodNamespace. Doesn't start with "PodNamespace"